### ISSUE TITLE
Hasn't been updated in months, outdated pre-compile.

### ISSUE THREAD

**Issue by slambo312** [100PXSquared/gmod-dxr#9]
Self-explanatory.

**Comment by Derpius** [comment]
Yeah working on gmod tracers in our community has kinda stagnated (me and https://github.com/MoxPoto got really far ahead and everyone else just gave up)

Falcor has been drastically updated since I last worked on this fully, so I'd need to essentially make a GModDXR2 to properly use all the new features, and I wanted to have BDPT (preferably metropolis light transport) up and running to use with it, but I never managed to get a working impl with the current version

I'm working on a language to replace E2/SF and a game at the moment (as well as having a full time job now) so my tracing has kinda fallen by the wayside, and without any competition there's no real reason to continue with it

There's precompiled binaries in releases you can use that have everything set up in the right directories, otherwise you'll need to change the Falcor submodule to the version I built it with (4.something, current version is 5), and there's a few lines that need changing inside of Falcor's code before compiling to remove the dependency on python

If I did ever make a more user friendly, upgraded tracer it'd probably be with Vulkan to remove the project's dependency on Windows, and remove all the extra dlls Falcor needs

**Comment by slambo312** [comment]
Alright.
